docs/v2/events/list_service_create_events.html

```diff
@@ -524,7 +524,9 @@ <h4>Body</h4>
           ],
           "documentation_url": null,
           "service_broker_guid": "3e0a6bd3-fb60-48cb-9929-18b4b95cbf7b",
-          "plan_updateable": false
+          "plan_updateable": false,
+          "instances_retrievable": false,
+          "bindings_retrievable": false,
```

Comment:
Should be no comma on line 529 -- json doesn't allow a comma before the close-brace.